flocker/acceptance/endtoend/test_cli.py

```diff
@@ -4,16 +4,103 @@
 Tests for the ``flocker-deploy`` command line tool.
 """
 
+from copy import deepcopy
+from subprocess import check_call
+
+from twisted.python.filepath import FilePath
 from twisted.trial.unittest import TestCase
+from yaml import safe_dump
+
+from ...control._config import FlockerConfiguration
+from ...control.httpapi import container_configuration_response
 
 from ..testtools import require_flocker_cli, require_cluster
-from ...testtools import random_name
+from ...testtools import loop_until, random_name
 
 
 class FlockerDeployTests(TestCase):
     """
     Tests for ``flocker-deploy``.
     """
+    def flocker_deploy(self, cluster, deployment_config, application_config):
+        """
+        Run ``flocker-deploy`` with given configuration files.
+
+        :param test_case: The ``TestCase`` running this unit test.
```

Comment:
Change the docstring `param` name and description to match the modified parameter.